frontend/src/components/Collaborators.vue

```diff
@@ -26,7 +26,8 @@
         </div>
 
         <input type="email" v-model="email" placeholder="name@domain.com"></input>
-        <button class="btn btn-success" v-on:click="addCollaborator()">Add</button>
+        <button class="btn btn-success" v-on:click="addCollaborator(false)">Add as Collaborator</button>
```

Comment:
I find this quite misleading that the collaborator one takes in a `false` while the Watcher is a `true`.

Reply:
I was going to have a button, where `v-on:click="readOnly = !readOnly"` and then in the addCollaborator function i just read that. But that does require more UI work.

Reply:
i can try doing that later. you mean a toggle button right?